pkg/urpc/urpc.go

```diff
@@ -72,7 +72,7 @@ type FilePayload struct {
 	Files []*os.File `json:"-"`
 }
 
-// ReleaseFD releases the indexth FD.
```

Comment:
nit: I think this was correct, though this isn't a real word. Perhaps it should be reworded "ReleaseFD releases the FD at the specified index."

Reply:
Ho right! I did not get it the first time. Done.